tests/system_tests/test_artifacts/test_data_types.py

```diff
@@ -122,3 +122,35 @@ def test_reference_table_artifacts(user, test_settings, wandb_backend_spy):
     run.log_artifact(art)
 
     run.finish()
+
+def test_table_mutation_logging(user, test_settings, wandb_backend_spy):
+    gql = wandb_backend_spy.gql
+    wandb_backend_spy.stub_gql(
+        gql.Matcher(operation="ServerInfo"),
+        gql.once(
+            content={
+                "data": {
+                    "serverInfo": {"cliVersionInfo": {"max_cli_version": "0.11.0"}}
```

Comment:
i believe it shouldn't be required, is the minimal server version we run doesn't support 0.11.0?